awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/Modals/NodeModals/NodeTypeStep/NodeTypeStep.jsx

```diff
@@ -47,9 +46,8 @@ function NodeTypeStep({
 }) {
   return (
     <>
-      <div css=" display: flex; align-items: center; margin-bottom: 20px;">
-        <b>{i18n._(t`Node Type`)}</b>
-        <VerticalSeperator />
+      <div css="display: flex; align-items: center; margin-bottom: 20px;">
+        <b css="margin-right: 24px">{i18n._(t`Node Type`)}</b>
```

Comment:
I would have this be done using a child selector once you update that component maybe